### ISSUE TITLE
Can't enable add-on in blender 3.5 ubuntu 22.04

### ISSUE THREAD

**Issue by RogerDass** [GPUOpen-LibrariesAndSDKs/BlenderUSDHydraAddon#272]
![image](https://user-images.githubusercontent.com/11430334/229319632-d9bc47d9-6b87-4db3-9a62-ebc7422627ad.png)
see image for corresponding error.

Note I have built and installed USD & MaterialX elsewhere, but I assume those included with this build are isolated?

**Comment by russkev** [comment]
I got the same error and it turned seemed that the plugin was picking up my external USD build. I deleted the path variables relating to USD and it seemed to solve this error, but then I ran into this: https://github.com/GPUOpen-LibrariesAndSDKs/BlenderUSDHydraAddon/issues/271

**Comment by russkev** [comment]
Actually I think this is an issue with Blender 3.5. I was able to get it working in Blender 3.4 (I'm in Windows though).

**Comment by RogerDass** [comment]
Hmmm…
I tried on both Ubuntu 22.04 and Windows 11. It very well could be a Blender 3.5 issue. I haven’t tried another version. I can’t say I’ve heard of “USDImagingLite” though.

**Comment by russkev** [comment]
Apparently it's internal to their build: https://github.com/GPUOpen-LibrariesAndSDKs/BlenderUSDHydraAddon/issues/33

**Comment by KarthikHebbarMN** [comment]
> Actually I think this is an issue with Blender 3.5. I was able to get it working in Blender 3.4 (I'm in Windows though).

Yes 3.5 on Windows also has compatibility issues with the plugin

**Comment by RogerDass** [comment]
Will there be any fix for this? Seems like there wasn't a new build of the plugin for quite some time...
Would love to add Blender and ProRender to the list of supported tools at Nike
